bundle/Controller/LocationController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzContentOnTheFlyBundle\Controller;
+
+use eZ\Publish\API\Repository\LocationService;
+use eZ\Publish\Core\REST\Server\Controller;
+use eZ\Publish\Core\REST\Server\Values;
+use Symfony\Component\HttpFoundation\Request;
+
+class LocationController extends Controller
+{
+    protected $locationService;
+
+    protected $configuration;
+
+    public function __construct(LocationService $locationService)
+    {
+        $this->locationService = $locationService;
+    }
+
+    public function suggestedAction(Request $request, $content)
+    {
+        if (!isset($this->configuration[$content]) && $content != 'default') {
+            $content = 'default';
+        }
+
+        if (isset($this->configuration[$content])) {
+            $locations = $this->configuration[$content]['location'];
+        } else {
+            $locations = [];
+        }
+
+        $suggested = [];
+        foreach ($locations as $locationId) {
+            $location = $this->locationService->loadLocation($locationId);
```

Comment:
Be aware this will throw if user does not have access to location, yyou should probably:
- try catch and skip those that throw UnAuthorized, you might want to do that on NotFound as well and in that case log the miss configuration using logger->warning
- We might also want to filter out locations user does not have create access in, which is $repo->canUser('content', 'create', $contentInfo, $location)

However we can do this when we move to config, as config we don't need to load things like child count for instance.

@bdunogier ?